cloud.ganttproject.colloboque/src/main/kotlin/cloud/ganttproject/colloboque/PostgreStorageApi.kt

```diff
@@ -112,6 +117,60 @@ class PostgreStorageApi(private val factory: ConnectionFactory, private val supe
     }
   }
 
+  fun parallelTransactions(
```

Comment:
I suggest the following signature:
`tryMergeConcurrentUpdates(connectionFactory, serverXlog, clientXlog)`

If my understanding is correct that at lines 131-132 we populate the temporary database/sceme with the data from `projectXml` then I suggest doing this before calling this method and pass a connection factory that connects to already initialized database.